apps/stable_diffusion/src/utils/stable_args.py

```diff
@@ -44,10 +44,7 @@ def is_valid_file(arg):
 p.add_argument(
     "--negative_prompts",
     nargs="+",
-    default=[
-        "watermark, signature, logo, text, lowres, ((monochrome, grayscale)), "
-        "blurry, ugly, blur, oversaturated, cropped"
-    ],
+    default=["trees", "green"],
```

Comment:
we can probably pass tree, green to our test on the command line and keep the other one for the default since it seems to make most visual impact.